<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="login_page">
    <widget class="QLabel" name="label_back_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>back3.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="PassWord_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PassWord</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw_line">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ID_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 13pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_line">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}

</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>430</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 13pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>join</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signup_page">
    <widget class="QLabel" name="label_back_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>back3.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="ID_label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="PassWord_label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PassWord</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_ok_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>490</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 15pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>join</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}

</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_NicName_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLabel" name="NicName_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PassWord Check</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_id_check">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>중복체크</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="room_list_page">
    <widget class="QLabel" name="label_back_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>back3.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>650</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 13pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>새로고침</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="room_name_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}

</string>
     </property>
    </widget>
    <widget class="QLabel" name="room_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>공백/특수문자를 제외한 방이름을 정해주세요</string>
     </property>
    </widget>
    <widget class="QListWidget" name="room_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>41</y>
       <width>281</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QList Widget
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QListWidget" name="online_users_list">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>191</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방 목록</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>접속중인 user</string>
     </property>
    </widget>
    <widget class="QPushButton" name="make_room_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>650</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 13pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>채팅 방 만들기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="room_page">
    <widget class="QLabel" name="label_back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>back3.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="room_chat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>610</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}

</string>
     </property>
    </widget>
    <widget class="QPushButton" name="room_send_btn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>620</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>입력</string>
     </property>
    </widget>
    <widget class="QListWidget" name="main_chat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>471</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QList Widget
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QListWidget" name="room_user_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QList Widget
{
    background-color:  rgb(255, 255, 255);
	border-radius: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방  user</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅 내용</string>
     </property>
    </widget>
    <widget class="QPushButton" name="room_exit_btn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>채팅방 나가기</string>
     </property>
    </widget>
    <widget class="QListWidget" name="room_online">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>친구 초대 (더블클릭)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="room_motion">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>660</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>모션 채팅</string>
     </property>
    </widget>
    <widget class="QPushButton" name="room_motion_close">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>660</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(245, 218, 43);
	border: 0px;
	border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string>모션채팅 끄기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="presonal_page">
    <widget class="QLabel" name="label_back_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/back3.jpg</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
